src/configure-nilrt-snac

```diff
@@ -185,7 +185,9 @@ EOF
 
 	# unload the modules if they are already loaded
 	# This will most likely error so send the output to /dev/null
+	set +e
 	rmmod cfg80211 mac80211 >/dev/null 2>&1
```

Comment:
Does `rmmod` return a different error code for a module that is not loaded, versus another failure to unload the module?

Reply:
The behavior is not documented, so the return code behavior should not be relied on.